main.py

```diff
@@ -0,0 +1,203 @@
+import json
+import requests
+import urllib
+import time
+from collections import defaultdict
+import email
+import imaplib
+import sys
+from pony import orm
+import quopri
+
+
+db = orm.Database()
+class Chats(db.Entity):
+    chat_id = orm.PrimaryKey(int)
+    state = orm.Required(int)
+    login = orm.Optional(str)
+    passwd = orm.Optional(str)
+db.bind(provider='sqlite', filename=':memory:')
+db.generate_mapping(create_tables=True)
+
+
+TOKEN = "5295625360:AAH5BaLlkwmC4iIRFbusIMSPREhaehbWykk"
+URL = "https://api.telegram.org/bot{}/".format(TOKEN)
+
+
+def get_url(url):
+    response = requests.get(url)
+    content = response.content.decode("utf8")
+    return content
+
+def get_json_from_url(url):
```

Comment:
2 blank lines (lower you have the same problem; PyCharm or sth like this may help you to follow PEP8)